Dockerfile

```diff
@@ -64,3 +64,7 @@ RUN cd /leonardo && \
 
 # Add Leonardo as a service (it will start when the container starts)
 CMD java $JAVA_OPTS -jar $(find /leonardo -name 'leonardo*.jar')
+
+# Install https://github.com/apangin/jattach to get access to JDK tools
```

Comment:
Nits:

* I'd put this before `CMD`, usually CMD is last
* Does `apt-get` work? Think that's slightly preferred in Dockerfiles